akka-stream/src/main/scala/akka/stream/impl/fusing/Interpreter.scala

```diff
@@ -9,6 +9,7 @@ import scala.util.control.NonFatal
 import akka.stream.stage._
 import akka.stream.Supervision
 import akka.stream.impl.ReactiveStreamsCompliance
+import akka.stream.FlowMaterializer
```

Comment:
This is still a layer violation that I don't like. This would prevent ripping out this class from Akka and reusing elsewhere, for example if a very lightweight stream support is needed somewhere without full akka dependencies. IteratorInterpreter is a good example since NoMaterializer seems more like a hack than a nice solution. I would rather take an Any, no matter how ugly it is, and cast it down at the use site, than to have this extra dependency.